ui/src/components/Row/Row.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from 'react';
```

Comment:
Nice job splitting up what was becoming a very large component.

However I think we need to rename this component something like LogsRow instead of just Row, because we will be doing the same thing fro the Stats component.